### ISSUE TITLE
Careful

### ISSUE THREAD

**Issue by AbleLincoln** [mightyselphie/prj-rev-bwfs-dasmoto#3]
Careful with your use of IDs. There can only be a single instance of any unique ID on a page. That means that no two elements can share the same ID. In your case, you have three spans with the ID of "price".  You should use a class for this.

https://github.com/mightyselphie/prj-rev-bwfs-dasmoto/blob/master/ArtsCrafts/index.html#L18

**Comment by mightyselphie** [comment]
yeah this one stumped me a bit. how should I have written the code with a class instead of an ID given that it is inline text?

**Comment by AbleLincoln** [comment]
You can still give a span a class. All elements can have both classes and IDs.
The code would look something like this:
`<span class="price">Starting at $3.00 / brush.</span>`

**Comment by mightyselphie** [comment]
Awesome! Super helpful :)
